src/main/java/application/SessionController.java

```diff
@@ -63,21 +76,46 @@ public SerializableSession getSession(@RequestParam(value="id") String id) {
 
 
     @RequestMapping("/answer_question")
-    public AnswerQuestionResponseBean answerQuestion(@RequestBody String body) throws Exception {
-        ObjectMapper mapper = new ObjectMapper();
-        AnswerQuestionRequestBean answerQuestionBean = mapper.readValue(body, AnswerQuestionRequestBean.class);
-        SerializableSession session = sessionRepo.find(answerQuestionBean.getSessionId());
-        FormEntrySession formEntrySession = new FormEntrySession(session);
-        JSONObject resp = AnswerQuestionJson.questionAnswerToJson(formEntrySession.getFormEntryController(),
-                formEntrySession.getFormEntryModel(),
-                answerQuestionBean.getAnswer(),
-                answerQuestionBean.getFormIndex());
-        session.setFormXml(formEntrySession.getFormXml());
-        session.setInstanceXml(formEntrySession.getInstanceXml());
-        sessionRepo.save(session);
-        System.out.println("Saving Session: " + session);
-        AnswerQuestionResponseBean responseBean = mapper.readValue(resp.toString(), AnswerQuestionResponseBean.class);
-        return responseBean;
+    public AnswerQuestionResponseBean answerQuestion(@RequestBody String body){
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            AnswerQuestionRequestBean answerQuestionBean = mapper.readValue(body, AnswerQuestionRequestBean.class);
+            SerializableSession session = sessionRepo.find(answerQuestionBean.getSessionId());
+            FormEntrySession formEntrySession = new FormEntrySession(session);
+            JSONObject resp = AnswerQuestionJson.questionAnswerToJson(formEntrySession.getFormEntryController(),
+                    formEntrySession.getFormEntryModel(),
+                    answerQuestionBean.getAnswer(),
+                    answerQuestionBean.getFormIndex());
+            session.setFormXml(formEntrySession.getFormXml());
+            session.setInstanceXml(formEntrySession.getInstanceXml());
+            sessionRepo.save(session);
+            AnswerQuestionResponseBean responseBean = mapper.readValue(resp.toString(), AnswerQuestionResponseBean.class);
+            return responseBean;
+        } catch (JsonParseException e) {
```

Comment:
i think this can be cut down to `JsonParseException|JsonMappingException|IOException e`. not blocking cause i think we're going to have a larger discussion about errors. in general we want to catch the errors we can handle/know how to better respond to them and do that. otherwise we'll let it fail hard and then have a handler return any unhandled exception (let's offline that). right now this just ignores some pretty severe errors and logs it to a file. realize my initial comment might've been misleading